src/styles/AchievementCommentCard.module.scss

```
@@ -0,0 +1,46 @@
+.question-header {
+  margin-top: 0;
+}
+
+.box-comment {
+  flex-grow: 1;
+  display: block;
+  padding-top: 0.1rem;
+  padding-left: 1rem;
+  padding-right: 1rem;
+
+  white-space: pre-wrap;
```

Stylelint (17.13.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)